### DISCUSSION TITLE
Clarification on Appwrite JWT Integration in Node.js Server with Client-Side Token Validation

### DISCUSSION THREAD

**Discussion by jostney** [appwrite/appwrite#7344] [Q&A]
### 👟 Reproduction steps

Aapologize for the length of my question but would appreciate your assistance in understanding the concept better.

We are in the process of designing our backend, and we plan to implement JWT validation. Each user will make requests by including a JWT, which our backend will then validate to ensure the authenticity of the user. On our Flutter client side, we have successfully implemented the creation of sessions and JWTs. However, we are facing challenges on the Node.js server side when handling certain pieces of information.

Currently, a new post is created on the server side using the following code. The client, along with its endpoint, projectId, and key values, are configured, and a global instance of "databases" is created:

```
const { config, attrs, enums } = require('./config');
const sdk = require("node-appwrite");
const client = new sdk.Client();

client.setEndpoint(config.endpoint);
client.setProject(config.projectId);
client.setKey(config.key);

const databases = new sdk.Databases(client);

const createPost = async (newPost) => {
    return await databases.createDocument(config.databaseId, config.collectionPostsId, sdk.ID.unique(), newPost);
}
```

Now, we want to incorporate JWT validation within the createPost method:
```
const createPost = async (token, newPost) => {
    // Validate the token with the SDK method here
    // ...

    return await databases.createDocument(config.databaseId, config.collectionPostsId, sdk.ID.unique(), newPost);
}
```

### 👍 Expected behavior

During my research, I came across the Appwrite documentation on JWT: [Appwrite JWT Documentation](https://appwrite.io/docs/products/auth/jwt). However, I find it challenging to grasp the concept, particularly the part where the JWT is set using the setJWT method:
```
const { Client } = require('node-appwrite');

const client = new Client()
    .setEndpoint('https://cloud.appwrite.io/v1') // Your API Endpoint
    .setProject('<PROJECT_ID>') // Your project ID
    .setJWT('eyJJ9.eyJ...886ca'); // Your secret JSON Web Token
```

### 👎 Actual Behavior

My confusion arises from the fact that user-specific information like JWT is set on an application-specific object (Client). I would like to understand how we can validate a user with their token in the given scenario:
```
const createPost = async (token, newPost) => {
    // Validate the token with the SDK method here
    // ...

    return await databases.createDocument(config.databaseId, config.collectionPostsId, sdk.ID.unique(), newPost);
}
```

### 🎲 Appwrite version

Appwrite Cloud

### 💻 Operating system

Something else

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
I would use separate `Client` instances just to make sure the permissions don't mix. So, you'll have one `Client` that acts as a user (via `client.setJWT()`) and another that has admin privileges (via `client.setKey()`). 

On the `Client` with the JWT, you can call `account.get()` to validate the token and get details about the user.
